features/inheritance/step_definitions.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+
+
+Given(/^a (Parent|Child) model with definition:$/) do |model, string|
+  if model == 'Parent'
```

Comment:
I did some trial and error of incorporating both models in a single step. It can work via implementing Data tables. Below is a sample of what it looks like. 
```
Given a parent and child model with definitions:
| ParentModel      | ChildModel |
| set_table_name('Animal') | boolean_attr :family_friendly |
| integer_attr  :id, hash_key: true ||
| string_attr :name, range_key: true ||
| string_attr :size ||
| list_attr :characteristics ||
| global_secondary_index(:gsi, hash_key: :id, range_key: :size, projection: {projection_type: "ALL"}) ||
```
It comes less readable so I'm going to opt with updating the step with `case` statement.